apps/condo/domains/ticket/components/IncidentHints.tsx

```diff
@@ -0,0 +1,219 @@
+import {
+    IncidentWhereInput,
+    Incident as IIncident,
+    IncidentClassifierIncidentWhereInput,
+    IncidentStatusType,
+    IncidentClassifier as IIncidentClassifier,
+    IncidentClassifierWhereInput,
+} from '@app/condo/schema'
+import { Col, Row } from 'antd'
+import dayjs from 'dayjs'
+import get from 'lodash/get'
+import React, { ComponentProps, useCallback, useEffect, useState } from 'react'
+
+import { useIntl } from '@open-condo/next/intl'
+import { Typography, Alert } from '@open-condo/ui'
+
+import {
+    IncidentProperty,
+    Incident,
+    IncidentClassifierIncident,
+} from '@condo/domains/ticket/utils/clientSchema'
+
+
+type Gutters = ComponentProps<typeof Row>['gutter']
+
+const INCIDENTS_GUTTER: Gutters = [0, 24]
+const DESCRIPTION_GUTTER: Gutters = [0, 14]
+
+type classifierDataType = Pick<IIncidentClassifier, 'category' | 'problem'>
+
+type IncidentHintsProps = {
+    propertyId: string
+    organizationId: string
+    classifier?: classifierDataType
+    dateISO?: string
+    onlyActual?: boolean
+}
+
+const formatDate = (date?: string) => {
+    if (!date) return '…'
+    return dayjs(date).format('DD.MM.YYYY HH.mm')
```

Comment:
You can use `intl.formatDate` method for that case

Reply:
It doesn't seem to be very customizable. And "dayjs" is mainly used for this